src/overlays/misc/ovl_map_mark_data/z_map_mark_data.c

```diff
@@ -1,5 +1,1457 @@
 #include "global.h"
 
+#if !OOT_MQ
+static MapMarkData sMapMarkDekuTree[] = {
+    // Deku Tree minimap 0
```

Comment:
I think it'd be nice to split the original/mq data in distinct files

For example have
`z_map_mark_data_original.inc.c`
`z_map_mark_data_mq.inc.c`

And in `z_map_mark_data.c` basically:

```c
#if !OOT_MQ
# include "z_map_mark_data_original.inc.c"
#else
# include "z_map_mark_data_mq.inc.c"
#endif
```

Reply:
Good idea, thanks

Reply:
What do you think of `z_map_mark_data_mq.c` and `z_map_mark_data_original.c` (or simply `z_map_mark_data.c`?) and include one or the other in the spec, similar to `z_message_PAL.c`? Right now VS Code doesn't like `z_lmap_mark_data_mq.c`, in complains it can't find `sMarkChestVtx`, which is a little annoying

Reply:
The reason I didn't suggest that initially is that would make both files be compiled for all versions, which can be confusing, but maybe it's not such a big deal

Also I personally don't like ifdefs in the spec because it's kind of hidden, but that's also fine

So yeah if you want